index.js

```diff
@@ -62,12 +49,25 @@ function processWarning () {
    * @property {STATE_CONSTANT} LIMITED_FINAL Indicates that the warning
    * is to be issued only once and has already been emitted.
    */
-  const STATES = {
-    UNLIMITED_INITIAL: 0,
-    UNLIMITED_ONGOING: -1,
-    LIMITED_INITIAL: 1,
-    LIMITED_FINAL: 2
-  }
+const STATES = {
+  UNLIMITED_INITIAL: 0,
+  UNLIMITED_ONGOING: -1,
+  LIMITED_INITIAL: 1,
+  LIMITED_FINAL: 2
+}
```

Comment:
can we have that as separate variables? Why an object?

Reply:
Just for grouping the things I think - can drop it